newton/_src/viewer/viewer_gl.py

```
@@ -316,6 +413,89 @@ def log_instances(self, name, mesh, xforms, scales, colors, materials, hidden=Fa
 
         self.objects[name].hidden = hidden
 
+    @override
+    def log_capsules(self, name, mesh, xforms, scales, colors, materials, hidden=False):
```

Ruff (0.14.14)

Unused method argument: `mesh`

(ARG002)